Tests/test_lifo_buffer.py

```diff
@@ -343,8 +341,8 @@ def test_process_neuron_list(self):
         self.lifo_buffer.set_answer("synapse 6 answer")
         with mock.patch("kalliope.core.TTS.TTSModule.generate_and_play"):
```

Comment:
This test is not passing.
I ran it in debug mode and the exception is well raised.
I don't know why the testing module doesn't catch it.
@LaMonF any idea?

Reply:
@Sispheor 
you may want to take a look [at this line](https://github.com/kalliope-project/kalliope/blob/8609a4e4daa50b6f60b32163dc2c43a9d4f19798/kalliope/core/Lifo/LIFOBuffer.py#L110) and changing "SynapseListAddedToLIFO" to "Exception". 
This way the test passed, at least therefore, I got another error now...

self.assertRaises seems not to catch the SynapseListAddedToLIFO exception, I guess this is why its not working. 

Also I'm not sure, but shouldn't it be like this:

```
        with mock.patch("kalliope.core.TTS.TTSModule.generate_and_play"):
            with self.assertRaises(SynapseListAddedToLIFO):
                  self.lifo_buffer._process_neuron_list(matched_synapse=matched_synapse)
```
? However, it works both ways...

Reply:
SynapseListAddedToLIFO extend Exception. So it should work the same way